
<launch>

  <param name="/filtre/zmax" value="1.9" type="double"/>
  <param name="/filtre/hue" value="0.0" type="double"/>
  <!--
  <param name="/filtre/hue" value="210.0" type="double"/>
  -->
  <param name="/filtre/delta_hue" value="20.0" type="double"/>

  <param name="/commande/max_curv" value="0.02" type="double" />
  <param name="/commande/plan_vel" value="0.2" type="double" />
  <param name="/commande/z_vel" value="1.0" type="double" />
  <param name="/commande/min_number" value="2000" type="int" />
  <param name="/commande/x_dev_min" value="0.1" type="double" />
  <param name="/commande/y_dev_min" value="0.1" type="double" />
  <param name="/commande/y_dev_min" value="0.1" type="double" />
  <param name="/commande/dz_dev_min" value="0.05" type="double" />

</launch>
